### PR TITLE
R4 medication request patient access documentation

### PR DESCRIPTION
**Pull request by khansasultana**
Description
----
R4 medication request patient access documentation

screenshot before merge
----
image
image
image
image

PR Checklist
----
- [ ] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/medications/medication-request.md [hunk 1]
```diff
@@ -133,12 +144,33 @@ We are currently always returning a value of "Order" in the intent field. We rec
...
 
 #### Response
 
 <%= headers status: 200 %>
 <%= json(:R4_MEDICATION_REQUEST_ENTRY) %>
+<%= disclaimer %>	
+
+#### Patient Authorization Request For Active Status	
+
+    GET https://fhir-ehr.devcerner.com/r4/2c400054-42d8-4e74-87b7-80b5bd5fde9f/MedicationRequest/262448077
```
Comment (aki-mathur) [inline content/millennium/r4/medications/medication-request.md:157]: These URLs should be of C1941 domain
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest

File: content/millennium/r4/medications/medication-request.md [hunk 2]
```diff
@@ -133,12 +144,33 @@ We are currently always returning a value of "Order" in the intent field. We rec
...
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:R4_MEDICATION_REQUEST_PATIENT_ENTRY) %>
+<%= disclaimer %>	
+
+#### Patient Authorization Request For Entered in Error Status	
+
+  GET https://fhir-ehr.devcerner.com/r4/2c400054-42d8-4e74-87b7-80b5bd5fde9f/MedicationRequest/261542609
```
Comment (aki-mathur) [inline content/millennium/r4/medications/medication-request.md:167]: These URLs should be of C1941 domain
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest

File: lib/resources/example_json/r4_examples_medication_request.rb [hunk 3]
```diff
@@ -148,6 +148,183 @@ module Resources
...
+          'timing': {
+            'repeat': {
+              'boundsPeriod': {
+                'start': '2020-12-08T21:58:00.000-08:00'
+              }
+            },
+            'code': {
+              'coding': [
+                {
+                  'system': 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/4003',
```
Comment (aki-mathur) [inline lib/resources/example_json/r4_examples_medication_request.rb:233]: These should point to c1941 tenant key

File: lib/resources/example_json/r4_examples_medication_request.rb [hunk 4]
```diff
@@ -148,6 +148,183 @@ module Resources
...
+                  'display': 'Once - dosing instruction fragment (qualifier value)'
+                }
+              ],
+              'text': 'Once'
+            }
+          },
+          'asNeededCodeableConcept': {
+            'coding': [
+              {
+                'system': 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/4005',
```
Comment (aki-mathur) [inline lib/resources/example_json/r4_examples_medication_request.rb:249]: Same as above

File: lib/resources/example_json/r4_examples_medication_request.rb [hunk 5]
```diff
@@ -148,6 +148,183 @@ module Resources
...
+                'code': '24199005',
+                'display': 'Feeling agitated (finding)'
+              }
+            ],
+            'text': 'agitation'
+          },
+          'route': {
+            'coding': [
+              {
+                'system': 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/4001',
```
Comment (aki-mathur) [inline lib/resources/example_json/r4_examples_medication_request.rb:264]: Same as above

File: lib/resources/example_json/r4_examples_medication_request.rb [hunk 6]
```diff
@@ -164,6 +341,3830 @@ module Resources
...
     }.freeze
 
+    R4_MEDICATION_REQUEST_PATIENT_BUNDLE ||= {
+      'resourceType': 'Bundle',
+      'id': '40d441d8-d53c-4bd6-b20e-d0c1af3244a3',
+      'type': 'searchset',
+      'link': [
+        {
+          'relation': 'self',
+          'url': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest?patient=12724065'
```
Comment (aki-mathur) [inline lib/resources/example_json/r4_examples_medication_request.rb:351]: These should be closed endpoints
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest

File: lib/resources/example_json/r4_examples_medication_request.rb [hunk 7]
```diff
@@ -164,6 +341,3830 @@ module Resources
...
+        },
+        {
+          'relation': 'next',
+          'url': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest?patient=12724065&-pageContext=T3BlblBsYXRmb3JtRmhpckNvbnRleHQ9dHJ1ZSZwYWdlQ29udGV4dD02MDY4MDUwXzYwNjgwNTFfMTI3MjQwNjVfMTI5MjMwOV8xJmNvbmNlcHQ9Y2hhcnRlZA%3D%3D&-pageDirection=NEXT'
```
Comment (aki-mathur) [inline lib/resources/example_json/r4_examples_medication_request.rb:355]: These should be closed endpoints
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest

File: lib/resources/example_json/r4_examples_medication_request.rb [hunk 8]
```diff
@@ -164,6 +341,3830 @@ module Resources
...
+        }
+      ],
+      'entry': [
+        {
+          'fullUrl': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest/314010287',
```
Comment (aki-mathur) [inline lib/resources/example_json/r4_examples_medication_request.rb:360]: These should be closed endpoints
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest

File: lib/resources/example_json/r4_examples_medication_request.rb [hunk 9]
```diff
@@ -164,6 +341,3830 @@ module Resources
...
+                'code': '{Capsule}'
+              }
+            },
+            'substitution': {
+              'allowedBoolean': true
+            }
+          }
+        },
+        {
+          'fullUrl': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest/314010037',
```
Comment (aki-mathur) [inline lib/resources/example_json/r4_examples_medication_request.rb:526]: These should be closed endpoints
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest

File: lib/resources/example_json/r4_examples_medication_request.rb [hunk 10]
```diff
@@ -164,6 +341,3830 @@ module Resources
...
+            'substitution': {
+              'allowedBoolean': true
+            },
+            'priorPrescription': {
+              'reference': 'MedicationRequest/313983017'
+            }
+          }
+        },
+        {
+          'fullUrl': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest/314974791',
```
Comment (aki-mathur) [inline lib/resources/example_json/r4_examples_medication_request.rb:692]: These should be closed endpoints
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest

File: lib/resources/example_json/r4_examples_medication_request.rb [hunk 11]
```diff
@@ -164,6 +341,3830 @@ module Resources
...
+                'start': '2020-08-12T08:48:57.000-05:00'
+              }
+            },
+            'priorPrescription': {
+              'reference': 'MedicationRequest/314010037'
+            }
+          }
+        },
+        {
+          'fullUrl': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest/311877117',
```
Comment (aki-mathur) [inline lib/resources/example_json/r4_examples_medication_request.rb:826]: These should be closed endpoints
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest

File: lib/resources/example_json/r4_examples_medication_request.rb [hunk 12]
```diff
@@ -164,6 +341,3830 @@ module Resources
...
+                'code': '{Capsule}'
+              }
+            },
+            'substitution': {
+              'allowedBoolean': true
+            }
+          }
+        },
+        {
+          'fullUrl': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest/314974793',
```
Comment (aki-mathur) [inline lib/resources/example_json/r4_examples_medication_request.rb:988]: These should be closed endpoints
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest

File: lib/resources/example_json/r4_examples_medication_request.rb [hunk 13]
```diff
@@ -164,6 +341,3830 @@ module Resources
...
+                'start': '2020-08-12T08:49:06.000-05:00'
+              }
+            },
+            'priorPrescription': {
+              'reference': 'MedicationRequest/311877117'
+            }
+          }
+        },
+        {
+          'fullUrl': 'https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest/311876937',
```
Comment (aki-mathur) [inline lib/resources/example_json/r4_examples_medication_request.rb:1122]: These should be closed endpoints
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/MedicationRequest

Comment (aki-mathur) [reply lib/resources/example_json/r4_examples_medication_request.rb:1122]: We need not have some many records

**Review by aki-mathur — CHANGES_REQUESTED** [review]

File: lib/resources/example_json/r4_examples_medication_request.rb [hunk 14]
```diff
@@ -148,6 +148,183 @@ module Resources
...
+            }
+          ]
+        }
+      ],
+      'priority': 'routine',
+      'reportedBoolean': false,
+      'medicationCodeableConcept': {
+        'coding': [
+          {
+            'system': 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/synonym',
```
Comment (ashu-bdvt) [inline lib/resources/example_json/r4_examples_medication_request.rb:186]: Should be 'system": 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/synonym'

File: lib/resources/example_json/r4_examples_medication_request.rb [hunk 15]
```diff
@@ -148,6 +148,183 @@ module Resources
...
+      },
+      'authoredOn': '2020-09-16T10:16:07.000-05:00',
+      'requester': {
+        'reference': 'Practitioner/605926',
+        'display': 'Carter, Kristen Cerner'
+      },
+      'courseOfTherapyType': {
+        'coding': [
+          {
+            'system': 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/4009',
```
Comment (ashu-bdvt) [inline lib/resources/example_json/r4_examples_medication_request.rb:208]: Should be 'system": 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/4009'

File: lib/resources/example_json/r4_examples_medication_request.rb (hunk 3 above)
Comment (ashu-bdvt) [reply lib/resources/example_json/r4_examples_medication_request.rb:233]: Should be 'system": 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/4003'

File: lib/resources/example_json/r4_examples_medication_request.rb (hunk 13 above)
Comment (ashu-bdvt) [reply lib/resources/example_json/r4_examples_medication_request.rb:1122]: Restrict bundle to have max 2 records

**Review by ashu-bdvt — CHANGES_REQUESTED** [review]

**Review by Aish1997 — APPROVED** [review]

File: content/millennium/r4/medications/medication-request.md (hunk 1 above)
Comment (khansasultana) [reply content/millennium/r4/medications/medication-request.md]: comment addressed

File: content/millennium/r4/medications/medication-request.md (hunk 2 above)
Comment (khansasultana) [reply content/millennium/r4/medications/medication-request.md]: comment addressed

File: lib/resources/example_json/r4_examples_medication_request.rb (hunk 3 above)
Comment (khansasultana) [reply lib/resources/example_json/r4_examples_medication_request.rb]: comment addressed

File: lib/resources/example_json/r4_examples_medication_request.rb (hunk 4 above)
Comment (khansasultana) [reply lib/resources/example_json/r4_examples_medication_request.rb]: comment addressed

File: lib/resources/example_json/r4_examples_medication_request.rb (hunk 5 above)
Comment (khansasultana) [reply lib/resources/example_json/r4_examples_medication_request.rb]: comment addressed

File: lib/resources/example_json/r4_examples_medication_request.rb (hunk 6 above)
Comment (khansasultana) [reply lib/resources/example_json/r4_examples_medication_request.rb]: comment addressed

File: lib/resources/example_json/r4_examples_medication_request.rb (hunk 7 above)
Comment (khansasultana) [reply lib/resources/example_json/r4_examples_medication_request.rb]: comment addressed

File: lib/resources/example_json/r4_examples_medication_request.rb (hunk 8 above)
Comment (khansasultana) [reply lib/resources/example_json/r4_examples_medication_request.rb]: comment addressed

File: lib/resources/example_json/r4_examples_medication_request.rb (hunk 9 above)
Comment (khansasultana) [reply lib/resources/example_json/r4_examples_medication_request.rb]: comment addressed

File: lib/resources/example_json/r4_examples_medication_request.rb (hunk 10 above)
Comment (khansasultana) [reply lib/resources/example_json/r4_examples_medication_request.rb]: comment addressed

File: lib/resources/example_json/r4_examples_medication_request.rb (hunk 11 above)
Comment (khansasultana) [reply lib/resources/example_json/r4_examples_medication_request.rb]: comment addressed

File: lib/resources/example_json/r4_examples_medication_request.rb (hunk 12 above)
Comment (khansasultana) [reply lib/resources/example_json/r4_examples_medication_request.rb]: comment addressed

File: lib/resources/example_json/r4_examples_medication_request.rb (hunk 13 above)
Comment (khansasultana) [reply lib/resources/example_json/r4_examples_medication_request.rb]: comment addressed

File: lib/resources/example_json/r4_examples_medication_request.rb (hunk 14 above)
Comment (khansasultana) [reply lib/resources/example_json/r4_examples_medication_request.rb]: comment addressed

File: lib/resources/example_json/r4_examples_medication_request.rb (hunk 15 above)
Comment (khansasultana) [reply lib/resources/example_json/r4_examples_medication_request.rb]: comment addressed

**Review by aki-mathur — APPROVED** [review]

**Review by ashu-bdvt — APPROVED** [review]

**Comment by VelocityKnight** [comment]
post merge screenshots:
Screen Shot 2020-12-17 at 4 23 45 PM
Screen Shot 2020-12-17 at 4 24 26 PM
Screen Shot 2020-12-17 at 4 24 35 PM
Screen Shot 2020-12-17 at 4 24 43 PM
Screen Shot 2020-12-17 at 4 24 51 PM
Screen Shot 2020-12-17 at 4 24 57 PM

**Pull request merged**